src/mkdocs_jupyter/nbconvert2.py

```diff
@@ -199,7 +200,11 @@ def nb2md(nb_path, start=0, end=None, execute=False, kernel_name=""):
         body, resources = exporter.from_file(nb_file)
     else:
         body, resources = exporter.from_filename(nb_path)
-    return body
+
+    # Code cells can also be created using backquotes (text surrounded by up to three backquotes `)
+    # So it needs to be removed, also to not mess the table of contents (for more see "test_toc.py")
+    backquote_text_regex = r"`{1,3}[.\s\S]*?`{1,3}"
```

Comment:
I fixed it by simply removing everything between 1-3 backquotes before file conversion, as done with the real code cells
https://github.com/danielfrg/mkdocs-jupyter/blob/1a1c220b2b10669dc9f197ecf20ccaff47f4c80f/src/mkdocs_jupyter/nbconvert2.py#L177-L178